### ISSUE TITLE
Air alarm sprites do not match their alarm status

### ISSUE THREAD

**Issue by msgerbs** [tgstation/tgstation#70039]
Reporting client version: 514.1585

## Testmerges:
- [TGUI for Techfabs II: The Great Recategorizing (AND ICONS) (AND MECHFABS) (AND AUTOLATHES)](https://www.github.com/tgstation/tgstation/pull/69990)
- [Multiz Rework: Human Suffering Edition (Contains PLANE CUBE)](https://www.github.com/tgstation/tgstation/pull/69115)
- [Removes canuserrotate from simple rotation's screentip](https://www.github.com/tgstation/tgstation/pull/70002)

## Reproduction:
Set off an air alarm (this is the easy part, amirite?). Reset the air alarm. Observe that even though the UI shows no area or local alarm, the sprite may still show the flashing orange or red alarm. Scream as you click the same air alarm for the 5th time that shift to see if it's actually alarming or not.

Does not really seem to be consistent, but it does seem to affect whole areas at once. Sometimes an area I saw this bug in would later show the correct status, sometimes it just stayed that way forever.

**Comment by Kitfox88** [comment]
air alarms and firelocks are servants of satan

**Comment by msgerbs** [comment]
Leave Satan out of this!

**Comment by Inari-Whitebear** [comment]
Yeah, you have to set and unset the atmos alarm on it for the sprite to update properly. Happens (especially?) in areas with multiple alarms. It's a pain.
